server/interceptor/interceptor_test.go

```diff
@@ -38,6 +38,23 @@ import (
 	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
 )
 
+func TestServiceName(t *testing.T) {
+	for _, tc := range []struct {
+		Desc       string
+		FullMethod string
+		Want       string
+	}{
+		{Desc: "grpc", FullMethod: "/trillian.TrillianLog/QueueLeaf", Want: "trillian.TrillianLog"},
```

Comment:
Maybe add tests for:
 - "/some.package.TrillianLog/Method" => "some.package.TrillianLog"
 - "noleading.slash.method" => ""

Reply:
done